### PR TITLE
Added latest version of CakeDC Users plugin (task #804)

### PR DESCRIPTION
**Pull request by georgeconstantinou**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
         "cakephp/migrations": "~1.0",
         "cakephp/plugin-installer": "*",
 		"qobo/phake-builder": "~1.0",
-        "holt59/cakephp3-bootstrap-helpers": "dev-master"
+        "holt59/cakephp3-bootstrap-helpers": "dev-master",
+        "cakedc/users": "3.1.*"
```
Comment (mamchenkov) [inline composer.json]: @georgeconstantinou it's better to use a "~" or "^" wildcard.  These two will make try to use the latest version, while the "*" will satisfy any version.  For example, if you have "3.1.5" installed and version "3.1.7" is available, then "3.1.*" will ignore it, while both "~" and "^" will try to upgrade to "3.1.7".

Comment (georgeconstantinou) [inline composer.json]: @mamchenkov `~3.1` and `^3.1` does not mean it will upgrade up to 4.0.0?

Comment (mamchenkov) [inline composer.json]: @georgeconstantinou nope ... have a look at this: 

https://getcomposer.org/doc/articles/versions.md#next-significant-release-operators

Comment (georgeconstantinou) [inline composer.json]: I see, less than `4.0.0`

**Comment by georgeconstantinou** [comment]
@mamchenkov please review

**Pull request merged**